v2/cmd/asoctl/pkg/importresources/importable_arm_resource.go

```diff
@@ -371,7 +371,8 @@ func (*importableARMResource) classifyError(err error) (string, bool) {
 
 		if responseError.StatusCode == http.StatusBadRequest {
 			if strings.Contains(responseError.Error(), "RequestUrlInvalid") ||
-				strings.Contains(responseError.Error(), "ValidationFailed") {
+				strings.Contains(responseError.Error(), "ValidationFailed") ||
+				strings.Contains(responseError.Error(), "NoRegisteredProviderFound") {
```

Comment:
This error could also in theory mean that the user hasn't registered this resource provider in their subscription? I guess even in that case the best thing we can do is just continue.